curriculum/challenges/english/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.english.md

```diff
@@ -31,7 +31,7 @@ Using the <code>type</code> attribute selector, try to give the checkboxes in Ca
 ```yml
 tests:
   - text: The <code>type</code> attribute selector should be used to select the checkboxes.
-    testString: assert(code.match(/<style>[\s\S]*?\[type=("|')checkbox\1\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi));
+    testString: assert(code.match(/<style>[\s\S]*?\[[\s\S]*?type[\s\S]*?=[\s\S]*?("|')checkbox\1[\s\S]*?\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi));
```

Comment:
Your current use of `[\s\S]*?` before `type`, after `type`, after `=`, and after `checkbox\1` would result in allowing extra non-space characters.  It would allow selectors like the following to pass the 1st test incorrectly;

```css
[types="checkbox"es] {

}
```
OR

```css
[funtype=shouldnotpass"checkbox"] {

}
```
See my suggestion of just using `\s*/` instead.
```suggestion
    testString: assert(code.match(/<style>[\s\S]*?\[\s*?type\s*?=\s*?("|')checkbox\1\s*?\]\s*?{[\s\S]*?}[\s\S]*?<\/style>/gi));
```